packages/grafana-ui/src/components/Link/TextLink.tsx

```diff
@@ -19,8 +19,8 @@ interface TextLinkProps extends Omit<AnchorHTMLAttributes<HTMLAnchorElement>, 't
   external?: boolean;
   /** True when the link will be displayed inline with surrounding text, false if it will be displayed as a block. Depending on this prop correspondant default styles will be applied */
   inline?: boolean;
-  /** The default variant is 'body'. To fit another styles set the correspondent variant as it is necessary also to adjust the icon size */
-  variant?: keyof ThemeTypographyVariantTypes;
+  /** The default variant is 'body'. To fit another styles set the correspondent variant as it is necessary also to adjust the icon size. `code` is excluded, as it is not fit for links. */
+  variant?: keyof Omit<ThemeTypographyVariantTypes, 'code'>;
```

Comment:
Just my initial thoughts, I can be convinced otherwise

Reply:
It seems like it makes sense, but just double checking with @staton-hyse11 what his opinion is. We can also remove this `Omit` later if we find a use case for it and it won't be a breaking change